docs/usage/gardener/gardener_info_configmap.md

```diff
@@ -0,0 +1,23 @@
+# Gardener Info `ConfigMap`
+
+## Overview
+
+The gardener apiserver maintains a [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/) inside the Garden cluster that contains information about the garden landscape.
```

Comment:
Nit: We do not follow this convention everywhere, but it looks better IMO.
```suggestion
The Gardener APIServer maintains a [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/) inside the Garden cluster that contains information about the Garden landscape.
```

Reply:
Following Kubernetes, it's either `gardener-apiserver` or Gardener API server.

See https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/ - just saying 😄

Reply:
Correct